source/_docs/automation/templating.markdown

```diff
@@ -3,7 +3,17 @@ title: "Automation Trigger Variables"
 description: "List all available variables made available by triggers."
 ---
 
-Automations support [templating](/docs/configuration/templating/) in the same way as scripts do. In addition to the [Home Assistant template extensions](/docs/configuration/templating/#home-assistant-template-extensions) available to scripts, the `trigger` template variable is available.
+Automations support [templating](/docs/configuration/templating/) in the same way as scripts do. In addition to the [Home Assistant template extensions](/docs/configuration/templating/#home-assistant-template-extensions) available to scripts, the `trigger`, and `this` template variables are available.
+
+The template variable `this` is also available when evaluating any `trigger_variables` declared in the configuration.
+
+## Available this Data
+
+| Template variable | Data |
+| ---- | ---- |
+| `this.entity_id` | The entity_id of the automation.
```

Comment:
missing `this.name`, `this.object_id`, `this.domain`, `this.last_changed`, `this.last_updated`, `this.context`

Reply:
+`this` is a state object. [State Objects](docs/configuration/state_object) provides a comprehensive description for the properties of `this` and `this.attributes`.

Reply:
Sure, but there are 3 listed. This now feels random or incomplete.

Reply:
OK I will remove the 3 and only leave the reference.